f3.go

```diff
@@ -47,6 +48,9 @@ type clientImpl struct {
 func (mc clientImpl) BroadcastMessage(ctx context.Context, mb *gpbft.MessageBuilder) error {
 	msg, err := mb.Build(mc.nn, mc.SignerWithMarshaler, mc.id)
 	if err != nil {
+		if errors.Is(err, gpbft.ErrNoPower) {
+			return nil
+		}
```

Comment:
@Kubuxu make sure the "no power" case makes sense.

Reply:
Yeah, it does.